.github/workflows/python-app-ubuntu-20.04.yml

```diff
@@ -0,0 +1,71 @@
+# This workflow will install Python dependencies, run tests and lint with a single version of Python
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: Python application
+
+on:
+  push:
+    branches: [ master, devel ]
+  pull_request:
+    branches: [ master, devel ]
+
+jobs:
+  build:
+    runs-on: ubuntu-20.04
+
+    steps:
+    - uses: actions/checkout@v3
+    - name: Prepare directory for packing with zip
+      run: |
+        mkdir -p out/deepness
+        cp -r plugin/deepness/ out/deepness/deepness
+    - name: Update
+      run: |
+        sudo apt update
+    - name: Install qgis
+      run: |
+        sudo apt install gnupg software-properties-common
+        sudo mkdir -m755 -p /etc/apt/keyrings
+        sudo wget -O /etc/apt/keyrings/qgis-archive-keyring.gpg https://download.qgis.org/downloads/qgis-archive-keyring.gpg
+        echo $'Types: deb deb-src\nURIs: https://qgis.org/debian\nSuites: focal\nArchitectures: amd64\nComponents: main\nSigned-By: /etc/apt/keyrings/qgis-archive-keyring.gpg' | sudo tee -a /etc/apt/sources.list.d/qgis.sources
```

Comment:
Here, you have to add the QGIS repository sources. It is required to obtain QGIS version 3.24+

Reply:
How about using the same action for both Ubuntu versions, but with a conditional step to avoid code duplication?  ( https://joshtronic.com/2020/05/23/only-run-step-for-specific-version-with-github-actions/ )

Reply:
Resolved